definitions/json/AuthorisationCaseField.json

```diff
@@ -587,18 +539,6 @@
     "UserRole": "et-acas-api",
     "CRUD": "R"
   },
-  {
-    "CaseTypeId": "ET_EnglandWales",
-    "CaseFieldID": "feeGroupReference",
```

Comment:
should potentially be kept as it's a way of identifying the case if the rep needs to contact the tribunal

Reply:
still removing this?